modules/cluster-logging-updating-logging.adoc

```diff
@@ -120,6 +120,7 @@ $ oc get cronjob
 [source,terminal]
 ----
 NAME                     SCHEDULE       SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+curator                  */5 * * * *    False     0        <none>          56s
```

Comment:
The default SCHEDULE is 30 3,9,15,21 * * *.
$ oc get cronjob
NAME                     SCHEDULE             SUSPEND   ACTIVE   LAST SCHEDULE   AGE
curator                  30 3,9,15,21 * * *   False     0        <none>          20s

Reply:
The other part /lgtm